src/network.cpp

```diff
@@ -21,18 +21,33 @@ This file is part of DarkStar-server source code.
 ===========================================================================
 */
 
+#include "helpers.h"
 #include "network.h"
+#include <iphlpapi.h>
+#include <vector>
 
 /* Externals */
 extern std::string g_ServerAddress;
```

Comment:
I think it might be about time to extract all of these sorts of things out into a config object which can be properly shared around the program, instead of using extern

Reply:
With a 15 second look, doing that would require some... interesting changes especially with some function hooks redirected into xiloader from ffxi referencing some global state. It looks like global state could reduced but not eliminated at a glance, perhaps the function hooks can add extra parameters into the redirected functions...?

That would be way outside the scope of this PR, but anonymous namespace is simple enough!